controllers/templatesync/template_sync.go

```diff
@@ -273,13 +286,27 @@ func (r *PolicyReconciler) Reconcile(ctx context.Context, request reconcile.Requ
 
 		var rsrc schema.GroupVersionResource
 
+		// Detect ConstraintTemplate
+		isGkConstraintTemplate := gvk.Group == utils.GvkConstraintTemplate.Group &&
+			gvk.Kind == utils.GvkConstraintTemplate.Kind
+		isGatekeeper := isGkConstraintTemplate || gvk.Group == utils.GConstraint
+
 		mapping, err := rMapper.RESTMapping(gvk.GroupKind(), gvk.Version)
 
 		if mapping != nil {
 			rsrc = mapping.Resource
 		} else {
 			resultError = err
-			errMsg := fmt.Sprintf("Mapping not found, please check if you have CRD deployed: %s", err)
+			errMsg := "Mapping not found, "
+			if isGatekeeper {
+				errMsg += "check if Gatekeeper is installed"
+				if gvk.Group == utils.GConstraint {
+					errMsg += " and the required " + utils.GvkConstraintTemplate.Kind + " has been deployed"
```

Comment:
```suggestion
					errMsg += " and the required ContraintTemplate for " + utils.GvkConstraintTemplate.Kind + " has been deployed"
```

Reply:
Yeah, I guess the kind isn't going to change much. 😆